tools/generator/Java.Interop.Tools.Generator.CodeGeneration/CodeGenerator.cs

```diff
@@ -109,7 +109,7 @@ public void WriteClass (ClassGen @class, string indent, GenerationInfo gen_info)
 			}
 
 			if (haveNested)
-				writer.WriteLine ("{0}\t}}\n", indent);
+				writer.WriteLine ("{0}\t}}{1}", indent, Environment.NewLine);
```

Comment:
This feels more like we should just have a separate `writer.WriteLine()` invocation and make the newline explicit.